### ISSUE TITLE
Wrong JDK version?

### ISSUE THREAD

**Issue by guikaua12** [DevNatan/inventory-framework#386]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### 🐛 Describe the bug

I'm using JDK 1.8 with IF 3.0.0 EAP
pom.xml:

```xml
<dependency>
    <groupId>me.devnatan</groupId>
    <artifactId>inventory-framework-bukkit</artifactId>
    <version>3.0.0-EAP-SNAPSHOT</version>
</dependency>
```

![image](https://github.com/DevNatan/inventory-framework/assets/33811154/13ae3b18-7b27-4560-aab5-3e308c351d56)

### ✔️ Expected behavior

Compile with no errors

### 👣 Steps to Reproduce

Compile with JDK 8

### 💻 Platform

- [X] Bukkit
- [ ] Sponge
- [ ] Other

### ⭐ Versions

MC 1.19.3, IF V3 EAP

**Comment by guikaua12** [comment]
Basically, i need to shade IF into my plugin, i don't want to depend on the plugin to be on the server

**Comment by rokrieger** [comment]
Any news or forecast?

**Comment by DevNatan** [comment]
Fixed in 3.0.0-alpha
